easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2023.10-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,480 @@
+        'checksums': ['a27a4a084d5e690e16e01e03ad2b2e552c61a65469419b907243193de1a84ae2'],
+    }),
+    ('sphinxcontrib_applehelp', '1.0.7', {
+        'modulename': 'sphinxcontrib.applehelp',
+        'checksums': ['39fdc8d762d33b01a7d8f026a3b7d71563ea3b72787d5f00ad8465bd9d6dfbfa'],
+    }),
+    ('sphinxcontrib_devhelp', '1.0.5', {
+        'modulename': 'sphinxcontrib.devhelp',
+        'checksums': ['63b41e0d38207ca40ebbeabcf4d8e51f76c03e78cd61abe118cf4435c73d4212'],
+    }),
+    ('sphinxcontrib_htmlhelp', '2.0.4', {
+        'modulename': 'sphinxcontrib.htmlhelp',
+        'checksums': ['6c26a118a05b76000738429b724a0568dbde5b72391a688577da08f11891092a'],
+    }),
+    ('sphinxcontrib-jsmath', '1.0.1', {
+        'modulename': 'sphinxcontrib.jsmath',
+        'checksums': ['a9925e4a4587247ed2191a22df5f6970656cb8ca2bd6284309578f2153e0c4b8'],
+    }),
+    ('sphinxcontrib_qthelp', '1.0.6', {
+        'modulename': 'sphinxcontrib.qthelp',
+        'checksums': ['62b9d1a186ab7f5ee3356d906f648cacb7a6bdb94d201ee7adf26db55092982d'],
+    }),
+    ('Babel', '2.13.1', {
+        'checksums': ['33e0952d7dd6374af8dbf6768cc4ddf3ccfefc244f9986d4074704f2fbd18900'],
+    }),
+    ('sphinx', '7.2.6', {
+        'checksums': ['9a5160e1ea90688d5963ba09a2dcd8bdd526620edbb65c328728f1b2228d5ab5'],
+    }),
+    ('sphinx-bootstrap-theme', '0.8.1', {
+        'checksums': ['683e3b735448dadd0149f76edecf95ff4bd9157787e9e77e0d048ca6f1d680df'],
+    }),
+    ('click', '8.1.7', {
+        'checksums': ['ca9853ad459e787e2192211578cc907e7594e294c7ccc834310722b41b9ca6de'],
+    }),
+    ('psutil', '5.9.6', {
+        'checksums': ['e4b92ddcd7dd4cdd3f900180ea1e104932c7bce234fb88976e2a3b296441225a'],
+    }),
+    ('future', '0.18.3', {
+        'checksums': ['34a17436ed1e96697a86f9de3d15a3b0be01d8bc8de9c1dffd59fb8234ed5307'],
+    }),
+    ('sortedcontainers', '2.4.0', {
+        'checksums': ['25caa5a06cc30b6b83d11423433f65d1f9d76c4c6a0c90e3379eaa43b9bfdb88'],
+    }),
+    ('intervaltree', '3.1.0', {
+        'checksums': ['902b1b88936918f9b2a19e0e5eb7ccb430ae45cde4f39ea4b36932920d33952d'],
+    }),
+    ('pytoml', '0.1.21', {
+        'checksums': ['8eecf7c8d0adcff3b375b09fe403407aa9b645c499e5ab8cac670ac4a35f61e7'],
+    }),
+    ('zipfile36', '0.1.3', {
+        'checksums': ['a78a8dddf4fa114f7fe73df76ffcce7538e23433b7a6a96c1c904023f122aead'],
+    }),
+    ('tomli_w', '1.0.0', {
+        'checksums': ['f463434305e0336248cac9c2dc8076b707d8a12d019dd349f5c1e382dd1ae1b9'],
+    }),
+    ('regex', '2023.10.3', {
+        'checksums': ['3fef4f844d2290ee0ba57addcec17eec9e3df73f10a2748485dfd6a3a188cc0f'],
+    }),
+    ('intreehooks', '1.0', {
+        'checksums': ['87e600d3b16b97ed219c078681260639e77ef5a17c0e0dbdd5a302f99b4e34e1'],
+    }),
+    ('pylev', '1.4.0', {
+        'checksums': ['9e77e941042ad3a4cc305dcdf2b2dec1aec2fbe3dd9015d2698ad02b173006d1'],
+    }),
+    ('pastel', '0.2.1', {
+        'source_tmpl': '%(name)s-%(version)s-py2.py3-none-any.whl',
```

Comment:
Just wondering if we shouldn't stick to using `SOURCE_WHL` / `SOURCE_PY3_WHL` for these wheel sources as in https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/p/Python-bundle-PyPI/Python-bundle-PyPI-2023.06-GCCcore-12.3.0.eb?

Reply:
we should, i just missed it (and inject checksums replaces these with the expanded template)